mcs/class/corlib/corlib.dll.sources

```diff
@@ -1594,21 +1594,31 @@ ReferenceSources/AppContextDefaultValues.cs
 ../referencesource/mscorlib/system/threading/waithandleExtensions.cs
 
+../../../external/corefx/src/Common/src/System/Threading/Tasks/RendezvousAwaitable.cs
+../../../external/corert/src/Common/src/System/Collections/Generic/LowLevelDictionary.cs
+../../../external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/DebuggerSupport.cs
+../../../external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/DebuggerSupport.Dummy.cs
+../../../external/corefx/src/Common/src/System/Threading/Tasks/BeginEndAwaitableAdapter.cs
+../../../external/corert/src/System.Private.CoreLib/shared/System/Threading/Tasks/TaskCanceledException.cs
 ../referencesource/mscorlib/system/threading/Tasks/AsyncCausalityTracer.cs
-../referencesource/mscorlib/system/threading/Tasks/BeginEndAwaitableAdapter.cs
-../referencesource/mscorlib/system/threading/Tasks/TaskFactory.cs
-../referencesource/mscorlib/system/threading/Tasks/FutureFactory.cs
-../referencesource/mscorlib/system/threading/Tasks/TaskScheduler.cs
-../referencesource/mscorlib/system/threading/Tasks/TaskCanceledException.cs
-../referencesource/mscorlib/system/threading/Tasks/Task.cs
-../referencesource/mscorlib/system/threading/Tasks/Future.cs
-../referencesource/mscorlib/system/threading/Tasks/TaskCompletionSource.cs
-../referencesource/mscorlib/system/threading/Tasks/TaskContinuation.cs
-../referencesource/mscorlib/system/threading/Tasks/TaskSchedulerException.cs
-../referencesource/mscorlib/system/threading/Tasks/Parallel.cs
-../referencesource/mscorlib/system/threading/Tasks/ParallelLoopState.cs
-../referencesource/mscorlib/system/threading/Tasks/ParallelRangeManager.cs
-../referencesource/mscorlib/system/threading/Tasks/ThreadPoolTaskScheduler.cs
+../../../external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/TaskFactory.cs
+../../../external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/FutureFactory.cs
+../../../external/corert/src/Common/src/System/Collections/Generic/LowLevelList.cs
+../../../external/corert/src/System.Private.Interop/src/InteropExtensions/Lock.cs
+../../../external/corert/src/System.Private.CoreLib/src/System/Threading/LockHolder.cs
+../../../external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/TaskScheduler.cs
+../../../external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/Task.cs
+../../../external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/Future.cs
+../../../external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/TaskContinuation.cs
+../../../external/corert/src/System.Private.CoreLib/shared/System/Threading/Tasks/TaskCompletionSource.cs
+../../../external/corert/src/System.Private.CoreLib/shared/System/Threading/Tasks/TaskSchedulerException.cs
+../../../external/corert/src/System.Private.CoreLib/src/System/Threading/Tasks/ThreadPoolTaskScheduler.cs
+../../../external/corefx/src/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/*.cs
```

Comment:
I don't like this, why do we need to pull whole System.Threading.Tasks.Parallel

Reply:
@marek-safar  Um but what is wrong with it? there are just a few files under this wildcard and I replaced old Parallel we had with corert's

Reply:
My error, I exchange it with some other parts